cw-plus/cw1/subkeys.md

```diff
@@ -0,0 +1,102 @@
+---
+order: 2
+---
+
+# Subkeys
+
+[CW1 Subkeys](https://github.com/CosmWasm/cosmwasm-plus/tree/master/contracts/cw1-subkeys)
+is inspired by [Cosmos SDK feature proposal](https://forum.cosmos.network/t/proposal-adding-subkey-feature-to-cosmos-sdk-and-apply-it-to-the-hub/2358).
+
+This is a basic proxy contract. Initiated contract with some tokens inside,
+allowed accounts can execute `CosmosMsg::Bank(BankMsg::Send{})` within the
+allowance(`spender, denom`) limits per account basis. Allowance logic is similar
+to [CW20](../cw20). After authorization their allowance will be reduced, and
+the send message will be relayed. If they don't have sufficient authorization,
+or if they try to proxy any other message type, then the attempt will be rejected.
+Admin can add an allowance for an account during init msg broadcast or after
+init.
+
+## Contract demo
+
+First, initialize node repl:
+
+```shell script
+npx @cosmjs/cli --init  https://raw.githubusercontent.com/CosmWasm/cosmwasm-plus/master/contracts/cw1-subkeys/helpers.ts
+```
+
+Load wallet:
+
+```ts
+const client = await useOptions(coralnetOptions).setup(PASSWORD);
+const factory = CW1(client);
+```
+
+Upload the code and the contract:
+
+```ts
+// contract is already uploaded on coralnet: codeId -> 12
+// const codeId = await factory.upload();
```

Comment:
Maybe remove some of the comments (the one explaining codeId -> 12 is good, the others confusing)